lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,12 @@
+module ActionController
```

Comment:
I'm not sure the `Context` class belongs in `ActionController`, as we might want to use it for non-AC related stuff as well. Some mixin/concern definitely belongs here, but I think the base `Context` class should be part of the `ActiveModel(::)Serializer(s)` namespace. Thoughts?